src/assets/scss/_multiLineEllipsis.scss

```diff
@@ -0,0 +1,32 @@
+// Taken from http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/
+
+/* mixin for multiline */
+@mixin multiLineEllipsis($lineHeight: 1.2em, $lineCount: 1, $bgColor: white){
```

Comment:
I'm really not a fan of sass mixins (but maybe that's just me?). This seems like a much overcomplicated fix for just adding the ellipsis. What's the benefit of having so many CSS rules and using pseudo selectors (like `.class:after`) here?

Reply:
The mixin is a pretty thin wrapper around straight CSS.  I don't think it's an issue here.

The .class:after is what adds the ellipsis.

Reply:
The article is an interesting read:

http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/